packages/elements/src/tree/elements/tree.ts

```diff
@@ -254,10 +268,136 @@ export class Tree<T extends TreeDataItem = TreeDataItem> extends List<T> {
    * @override
    */
   protected update (changedProperties: PropertyValues): void {
+    super.update(changedProperties);
```

Comment:
In my view, in most cases `super` should be called before performing tasks because the `super class` might have some performing tasks e.g. initial, prepare something first so if we call `super` after, the `super class` a component might be a break in some cases.